src/App.tsx

```diff
@@ -1,44 +1,15 @@
 import React from 'react';
 import './App.css';
 import { H1 } from './styles/typography/h1';
-import { H2 } from './styles/typography/h2';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faHeart } from '@fortawesome/free-solid-svg-icons';
-import { Button } from './components/buttons/button';
+import { faCar } from '@fortawesome/free-solid-svg-icons';
+import { PropertyInfo } from './components/property-info/property';
 
 function App() {
   return (
     <div className="App">
       <H1> Styleguide </H1>
-      <H2> Botões </H2>
-      <Button type="primary" icon={<FontAwesomeIcon icon={faHeart} />}>
-        primary button
-      </Button>
-      <div className={'spacing'}></div>
-      <Button type="secondary">secondary button</Button>
-      <div className={'spacing'}></div>
-      <Button type="cta">Ctabutton</Button>
-      <div className={'spacing'}></div>
-      <Button type="primary" icon={<FontAwesomeIcon icon={faHeart} />} compact>
-        primary button
-      </Button>
-      <div className={'spacing'}></div>
-      <Button type="secondary" compact>
-        secondary button
-      </Button>
-      <div className={'spacing'}></div>
-      <Button type="cta" compact>
-        Ctabutton
-      </Button>
-      <div className={'spacing'}></div>
-      <Button type="primary" disable>
-        primary button
-      </Button>
-      <div className={'spacing'}></div>
-      <Button type="primary" disable compact>
-        primary button
-      </Button>
-      <div className={'spacing'}></div>
+      <PropertyInfo title="Carro" icon={<FontAwesomeIcon icon={faCar} />}></PropertyInfo>
```

Comment:
uma coisa que costumamos fazer é modulariar esses componentes que vêm de apis/libs externas para caso em algum momento quisermos mudar a fonte dos dados, seria facil pois estaria num só arquivo. Se quiser aplicar aqui, pode ser uma boa. Fica a sugestão:


```typescript
import { faCar } from '@fortawesome/free-solid-svg-icons';
import { FontAwesomeIcon, FontAwesomeIconProps } from '@fortawesome/react-fontawesome';

type MyFontAwesomeIconProps = Omit<FontAwesomeIconProps, 'icon'>;

export const FaIcon = {
  Car: (props: MyFontAwesomeIconProps) => <FontAwesomeIcon icon={faCar} {...props} />,
};
```

A implementacao fica assim:
```suggestion
      <PropertyInfo title="Carro" icon={<FaIcon.Car/>} />
```
Conforme for adicionando novos icones, da para alimentar o objeto do FaIcon com esses novos e assim vai... o que acha?

Reply:
Uma segunda coisa que nao tem nada a ver com a primeira é que quando vc tem esses componente que não recebe nenhum filho, vc pode usar um [self-closing tag](https://react-cn.github.io/react/tips/self-closing-tag.html)